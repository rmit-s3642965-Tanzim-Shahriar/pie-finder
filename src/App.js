import React from 'react';
import './App.scss';
import MainHeader from './components/mainHeader';
import CustomFooter from './components/customFooter';
import PieList from './components/pieList';
import store from './store'

function App() {
  return (
    // <Provider store={store}>
    <div className="App">
    <MainHeader/>
    <PieList/>
    <CustomFooter/>
  </div>
  // </Provider>
  );
}

export default App;
