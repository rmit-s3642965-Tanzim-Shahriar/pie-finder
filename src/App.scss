.App {
  text-align: center;
  color:white;
  display:flex;
  flex-direction: column;
}

