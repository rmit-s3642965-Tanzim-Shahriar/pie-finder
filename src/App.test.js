import React from 'react';
import { mount } from 'enzyme';
import App from './App';
//WARNING: MUST comment out import .scss files from App.js, MainContent.js, About.js, ListOfStores.js. 
//Lines like import './App.scss' should be commented out as jest cant read .scss files
it('renders and mounts app without crashing', () => {
  const component = mount(<App />);
  expect(component).toMatchSnapshot();
});