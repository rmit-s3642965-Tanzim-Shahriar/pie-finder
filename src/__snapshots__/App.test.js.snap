// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders and mounts app without crashing 1`] = `
<App>
  <Provider
    store={
      Object {
        "dispatch": [Function],
        "getState": [Function],
        "replaceReducer": [Function],
        "subscribe": [Function],
        Symbol(observable): [Function],
      }
    }
  >
    <div
      className="App"
    >
      <ConnectFunction>
        <MainContent
          error={Array []}
          fetchPies={[Function]}
          fetchStores={[Function]}
          loaded={
            Object {
              "pieLoaded": false,
              "storeLoaded": false,
            }
          }
          pies={Array []}
          stores={Array []}
        >
          <div
            className="MainContent"
          >
            <div
              className="loading"
            >
              <div>
                Please Wait, Loading
              </div>
              <div>
                <WithStyles(CircularProgress)>
                  <CircularProgress
                    classes={
                      Object {
                        "circle": "MuiCircularProgress-circle-7",
                        "circleDisableShrink": "MuiCircularProgress-circleDisableShrink-10",
                        "circleIndeterminate": "MuiCircularProgress-circleIndeterminate-9",
                        "circleStatic": "MuiCircularProgress-circleStatic-8",
                        "colorPrimary": "MuiCircularProgress-colorPrimary-4",
                        "colorSecondary": "MuiCircularProgress-colorSecondary-5",
                        "indeterminate": "MuiCircularProgress-indeterminate-3",
                        "root": "MuiCircularProgress-root-1",
                        "static": "MuiCircularProgress-static-2",
                        "svg": "MuiCircularProgress-svg-6",
                      }
                    }
                    color="primary"
                    disableShrink={false}
                    size={40}
                    thickness={3.6}
                    value={0}
                    variant="indeterminate"
                  >
                    <div
                      className="MuiCircularProgress-root-1 MuiCircularProgress-colorPrimary-4 MuiCircularProgress-indeterminate-3"
                      role="progressbar"
                      style={
                        Object {
                          "height": 40,
                          "width": 40,
                        }
                      }
                    >
                      <svg
                        className="MuiCircularProgress-svg-6"
                        viewBox="22 22 44 44"
                      >
                        <circle
                          className="MuiCircularProgress-circle-7 MuiCircularProgress-circleIndeterminate-9"
                          cx={44}
                          cy={44}
                          fill="none"
                          r={20.2}
                          strokeWidth={3.6}
                          style={Object {}}
                        />
                      </svg>
                    </div>
                  </CircularProgress>
                </WithStyles(CircularProgress)>
              </div>
            </div>
          </div>
        </MainContent>
      </ConnectFunction>
    </div>
  </Provider>
</App>
`;
